be/btree.h

```diff
@@ -563,8 +563,8 @@ M0_INTERNAL void m0_be_btree_release(struct m0_be_tx           *tx,
  * Used for cursor depth-first in-order traversing.
  */
 struct m0_be_btree_cursor_stack_entry {
-	struct m0_be_bnode *bs_node;
 	int                 bs_idx;
+	struct m0_be_bnode *bs_node;
```

Comment:
What is the reason for this change of variables ?

Reply:
To compact the structure.

Reply:
Can you explain more here?
The size of this struct is:
int,        8 bytes, 64 bits
pointer, 8 bytes, 64 bits